exercises/pangram/pangram_test.exs

```diff
@@ -0,0 +1,60 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("pangram.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule PangramTest do
+  use ExUnit.Case
+
+  # @tag :pending
+  test "empty sentence" do
+    refute Pangram.pangram?("")
+  end
+
+  @tag :pending
+  test "pangram with only lower case" do
+    assert Pangram.pangram?("the quick brown fox jumps over the lazy dog")
+  end
+
+  @tag :pending
+  test "missing character 'x'" do
+    refute Pangram.pangram?("a quick movement of the enemy will jeopardize five gunboats")
+  end
+
+  @tag :pending
+  test "another missing character 'x'" do
+    refute Pangram.pangram?("the quick brown fish jumps over the lazy dog")
+  end
+
+  @tag :pending
+  test "pangram with underscores" do
+    assert Pangram.pangram?("the_quick_brown_fox_jumps_over_the_lazy_dog")
+  end
+
+  @tag :pending
+  test "pangram with numbers" do
+    assert Pangram.pangram?("the 1 quick brown fox jumps over the 2 lazy dogs")
+  end
+
+  @tag :pending
+  test "missing letters replaces by numbers" do
+    refute Pangram.pangram?("7h3 qu1ck brown fox jumps ov3r 7h3 lazy dog")
+  end
+
+  @tag :pending
+  test "pangram with mixed case and punctuation" do
+    assert Pangram.pangram?("Five quacking Zephyrs jolt my wax bed.")
+  end
+
+  @tag :pending
+  test "pangram with non ascii characters" do
+    assert Pangram.pangram?("Five quacking Zephyrs jolt my wax bed.")
```

Comment:
Looks like the input for this test should be 

`Victor jagt zwölf Boxkämpfer quer über den großen Sylter Deich.`